bzt/engine.py

```diff
@@ -1063,6 +1066,17 @@ def __init__(self):
         self.start_time = None
         self.preprocess_args = lambda x: None
 
+        self.features = {
```

Comment:
This seems a bit unjustified at the moment, but if we can find more executor features to be used through this interface — i'm all for it.

Reply:
+1